networks/stagenet/clearnet.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
`clearnet` name located inside `stagenet` creates an impression that it should be located on the same level as `stagenet`. Maybe better name for this file can be `blockchain.json` or `network.json`.

Reply:
I agree, but in the same time it should somehow tell that here storing both chain info and clear sync contracts addresses

Reply:
Then `blockchain.json` should be fine